backend/src/validators/contracts.js

```diff
@@ -20,7 +20,7 @@ const getAll = {
 
 const getOne = {
   params: Schema.IdParam,
-  response: getResponse(body),
+  //response: getResponse(body),
```

Comment:
Are you planning on updating this once it becomes more established?

Reply:
Yeah, I'm working on proper validation now but it wasn't ready yet in this PR.